tests/unit/build-order-actions.spec.js

```diff
@@ -164,12 +176,12 @@ test(`lineItems
     },
   )
 
-  const actions = buildOrderActions.lineItems(order)
+  const actions = buildOrderActions.lineItems(order, orderSample())
```

Comment:
the `orderSample` object is used in several places, modifying the object in one place affect every other place the object is being used. So the _best_ way to avoid this is to instantiate a new object every time better than cloning the object 👼